docs/en/query_language/functions/array_functions.md

```diff
@@ -670,31 +690,72 @@ SELECT arrayDifference([1, 2, 3, 4])
 
 Example of the overflow due to result type Int64:
 
+Query:
+
 ```sql
 SELECT arrayDifference([0, 10000000000000000000])
 ```
 
+Result:
+
 ```text
 ┌─arrayDifference([0, 10000000000000000000])─┐
 │ [0,-8446744073709551616]                   │
 └────────────────────────────────────────────┘
 ```
 
-## arrayDistinct(arr) {#array_functions-arraydistinct}
+## arrayDistinct {#arraydistinct}
 
-Takes an array, returns an array containing the distinct elements. 
+Takes an array, returns an array containing unique elements only.
```

Comment:
The term "the distinct elements" sounds more accurate to me.

Considering:
`[1,2,2,2,3,3,4]` -> `[1,2,3,4]`
vs.
`[1,2,2,2,3,3,4]` -> `[1,4]`

Reply:
Done